@uportal/eyebrow-user-info/package.json

```diff
@@ -1,14 +1,18 @@
 {
   "name": "@uportal/eyebrow-user-info",
-  "version": "1.36.0",
+  "version": "1.37.0",
```

Comment:
:thinking: I'm surprised this wasn't updated, the release already appears to be on npm https://www.npmjs.com/package/@uportal/eyebrow-user-info/v/1.37.0
/cc @bjagg is there an extra commit from the last release which could be pushed?